src/db/cli.ts

```diff
@@ -12,4 +12,6 @@ export const dataSource = new DataSource({
     get: (key: string, def?: string) => process.env[key] || def,
   }),
   type: 'postgres',
+  installExtensions: true,
```

Comment:
Attempt to fix https://github.com/energywebfoundation/ssi-hub/actions/runs/4066246957/jobs/7002070873#step:11:161

Reply:
is this still necessary?

Reply:
oh, nevermind. it's outdated :)

Reply:
sorry, forget to rm comment. will rm now